homeassistant/components/airzone/select.py

```diff
@@ -0,0 +1,160 @@
+"""Support for the Airzone sensors."""
+from __future__ import annotations
+
+from dataclasses import dataclass, replace
+from typing import Any, Final
+
+from aioairzone.common import GrilleAngle, SleepTimeout
+from aioairzone.const import (
+    API_COLD_ANGLE,
+    API_HEAT_ANGLE,
+    API_SLEEP,
+    AZD_COLD_ANGLE,
+    AZD_HEAT_ANGLE,
+    AZD_NAME,
+    AZD_SLEEP,
+    AZD_ZONES,
+)
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AirzoneUpdateCoordinator
+from .entity import AirzoneEntity, AirzoneZoneEntity
+
+
+@dataclass
+class AirzoneSelectDescriptionMixin:
+    """Define an entity description mixin for select entities."""
+
+    api_param: str
+    options_dict: dict[str, int]
+
+
+@dataclass
+class AirzoneSelectDescription(SelectEntityDescription, AirzoneSelectDescriptionMixin):
+    """Class to describe an Airzone select entity."""
+
+
+GRILLE_ANGLE_DICT: Final[dict[str, int]] = {
```

Comment:
Might be nice to make these translatable in a future PR but since they are mostly numbers I'm not sure its worth it

ex
https://github.com/home-assistant/core/blob/a5225a360662ed2465d04835281d3d2e2dca2604/homeassistant/components/sensibo/strings.json#L49